### PR TITLE
build against gnome runtime

### LINKED ISSUE #30: Update Webkitgtk to v2.25.4

### PR DESCRIPTION
**Pull request by abderrahim**
This allows to drop quite a few modules.

This isn't ready for merge yet as it currently uses the unmerged i386 compat extensions from https://gitlab.gnome.org/GNOME/gnome-build-meta/merge_requests/338

Hopefully, we'll get `3.34beta` of gnome on flathub beta later this week and we can then switch to it.

This should fix #30 (and remove the need to worry about it in the future)

### CODE REVIEW

File: net.lutris.Lutris.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id: net.lutris.Lutris
-sdk: org.freedesktop.Sdk
-runtime: org.freedesktop.Platform
-runtime-version: "19.08beta"
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: master
```
Comment (gasinvein) [inline net.lutris.Lutris.yml]: Please change `master` to the some branched version once it's available to install.

**Comment by gasinvein** [comment]
@abderrahim Will `3.34beta` land on flathub-beta?

**Comment by barthalion** [comment]
↑ It's me, sorry for the noise.

**Comment by LeandroStanger** [comment]
bot, build net.lutris.Lutris

File: net.lutris.Lutris.yml (hunk 1 above)
Comment (abderrahim) [reply net.lutris.Lutris.yml]: Changed to 3.34beta

File: net.lutris.Lutris.yml [hunk 2]
```diff
@@ -59,7 +59,7 @@ add-extensions:
     merge-dirs: bin
 
 sdk-extensions:
-  - org.freedesktop.Sdk.Compat.i386
+  - org.gnome.Sdk.Compat.i386
   - org.freedesktop.Sdk.Extension.toolchain-i386
```
Comment (gasinvein) [inline net.lutris.Lutris.yml]: @abderrahim AFAIK the `toolchain-i386` extension is built against FD.o SDK. Will it keep working on Gnome SDK just as fine?

Comment (abderrahim) [reply net.lutris.Lutris.yml]: Yes, it should. We inherit all the sdk extensions from freedesktop-sdk.

**Comment by gasinvein** [comment]
Great, thanks @abderrahim for your effort.

**Pull request merged**